### ISSUE TITLE
[Frontend] Same view on page refresh (Prosjekter)

### ISSUE THREAD

**Issue by trondoe** [Puzzlepart/did#418]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
When doing a page refresh (F5) the same view should be displayed afterwards.

### Description
When I'm at the Prosjekter>MineProsjekter view and have made a change in my calendar and do a refresh of the view by performing a page refresh (F5) in the browser, the view changes to Søk. See screenshots below. When performing a page refresh, it makes sense to keep the active view after the refresh. This issue is related to https://github.com/Puzzlepart/did365/issues/415

Before reresh:
![image](https://user-images.githubusercontent.com/8970087/83021591-83e33e80-a02a-11ea-8a87-127a58573441.png)

After reresh:
![image](https://user-images.githubusercontent.com/8970087/83021633-95c4e180-a02a-11ea-91fa-8815f187ab8c.png)
